packages/domain-search/src/components/DomainsFullCart/DomainsFullCart.tsx

```diff
@@ -1,26 +1,106 @@
+import {
+	// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+	__unstableMotion as motion,
+	Card,
+	CardHeader,
+	__experimentalHeading as Heading,
+	CardBody,
+	Button,
+	CardFooter,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { close } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import { useCallback, useRef, useEffect } from 'react';
 import { useDomainSearch } from '../DomainSearch/DomainSearch';
 import { DomainsFullCartItems } from './Items';
+import { DomainsFullCartSummary } from './Summary';
+
+import './style.scss';
 
 const DomainsFullCart = ( { children }: { children?: React.ReactNode } ) => {
-	const { isFullCartOpen } = useDomainSearch();
+	const { isFullCartOpen, closeFullCart, onContinue } = useDomainSearch();
+	const { __ } = useI18n();
+	const isMobile = ! useViewportMatch( 'small' );
+
+	const fullCartRef = useRef< HTMLDivElement >( null );
+
+	const handleClickOutside = useCallback(
+		( event: MouseEvent ) => {
+			if ( fullCartRef.current && ! fullCartRef.current.contains( event.target as Node ) ) {
+				closeFullCart();
+			}
+		},
+		[ closeFullCart ]
+	);
 
-	if ( ! isFullCartOpen ) {
-		return null;
-	}
+	useEffect( () => {
+		if ( isFullCartOpen ) {
+			document.addEventListener( 'mousedown', handleClickOutside );
+			return () => {
+				document.removeEventListener( 'mousedown', handleClickOutside );
+			};
+		}
+	}, [ isFullCartOpen, handleClickOutside ] );
 
 	return (
-		<div
-			style={ {
-				position: 'fixed',
-				top: 0,
-				right: 0,
-				bottom: 0,
-				background: 'blue',
-			} }
+		<motion.div
+			ref={ fullCartRef }
+			initial={
+				isMobile
+					? { y: '100%', display: 'none', opacity: 0 }
+					: { x: '100%', display: 'none', opacity: 0 }
+			}
+			animate={
+				isMobile
+					? {
+							y: isFullCartOpen ? 0 : '100%',
+							display: isFullCartOpen ? 'block' : 'none',
+							opacity: isFullCartOpen ? 1 : 0,
+					  }
+					: {
+							x: isFullCartOpen ? 0 : '100%',
+							display: isFullCartOpen ? 'block' : 'none',
+							opacity: isFullCartOpen ? 1 : 0,
+					  }
+			}
+			transition={ { type: 'tween', duration: 0.25 } }
```

Comment:
I thought that this pattern worked well for this case, but let me know if we want to do it differently.